tox.ini

```diff
@@ -77,9 +77,20 @@ commands =
 
 [testenv:lint]
 description = Performs linting, style checks
-skip_install = true
-sitepackages = false
+skip_install = false
+usedevelop = True
+allowlist_externals =
+    bash
+extras =
+    azure
+    docker
+    ec2
+    gce
+    podman
+    vagrant
 deps =
+    py-{devel}: git+https://github.com/ansible-community/molecule.git@main#egg=molecule[test]
```

Comment:
No need to add this line, I think.

It is defined in `[testenv]` already.

Reply:
DONE